cmd/create/cluster/cmd.go

```diff
@@ -1179,6 +1179,7 @@ func run(cmd *cobra.Command, _ []string) {
 			Help:     cmd.Flags().Lookup("version").Usage,
 			Options:  versionList,
 			Default:  version,
+			Required: true,
```

Comment:
Does this change related to OCM-4646?

Reply:
This one is required due to the changes to how the options prompt works. So that we may not skip the version selection as it was before